_pages/en_US/finalizing-setup.txt

```diff
@@ -67,7 +67,7 @@ If, before following this guide, you already had an EmuNAND setup and would like
   + Updates while using B9S + Luma (what you have) are safe
   + The updater may display a message saying "Your system is up to date" instead of updating. This is normal if you are already up to date; continue with the next section
   + If this gives you an error, set your DNS settings to "auto"
-  + If this still gives you an error, [follow CTRTransfer](ctrtransfer), then try updating again
+  + If this still gives you an error, [follow this troubleshooting guide](troubleshooting#wifi-connection-errors-after-installing-cfw), then try updating again
```

Comment:
You are suggesting multiple ideas, each of which should be tried before updating again. This isn't necessarily clear from the text of your troubleshooting guide.